build/build-lr-files.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
do you need a place to copy files from the external repo to deploy for LR?

how about doing it like `build-lightrider.js`, analogous to `build-extension.js`?
- Have it build the report generator, put it in `dist/lightrider/report-generator.js`
- call [`buildBundle` to build the bundle](https://github.com/GoogleChrome/lighthouse/blob/a95fd1d0be04a9aabbc53c65ba0efd5883f5d087/build/build-extension.js#L26-L30) instead of doing it in package.json, save to `dist/lightrider/lighthouse-lr-bundle.js`
- copy all the report renderer files needed for upload into `dist/lightrider/resources/` or `report-resources/` or whatever

then everything needed is all in one place for copying to the various places they need to be copied

Reply:
Love it.  Done.